test/testcases/errors-and-warnings.bats

```diff
@@ -31,9 +31,9 @@
   [[ true ]]
 }
 
-@test "Exits with non zero when an unknown command is used" {
-    run ./bin/run makemeasandwich --with cheese
-    [[ $status -ne 0 ]]
+@test "Suggest similar commands if unknown command is used" {
+   output=echo "n" | ./bin/run helpp 2>&1 | grep -q "Did you mean help"
+   [[ $output -eq 0 ]]
```

Comment:
What happens when a command that isn't similar to any existing command is used? We still need a test for that.

Reply:
it's [not configurable](https://github.com/oclif/plugin-not-found/blob/main/src/index.ts#L15) sadly. I kind of expected it to discard suggestions where dist > N, but no

Reply:
Okay, cool 👍 Let's merge this!